### PR TITLE
[RFR] replaced force_navigate calls with navigate_to

### PR DESCRIPTION
**Pull request by izapolsk**
Purpose or Intent
=================
1. replaced force_navigate in several tests

{{pytest: --use-provider=vsphere55 cfme/tests/webui/test_splitter.py cfme/tests/intelligence/reports/test_menus.py }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit a46925d721b5016a15cd2ac38990d790c929c687:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/tests/webui/test_splitter.py [hunk 1]
```diff
@@ -1,20 +1,44 @@
...
+def nav_to(location):
+    if isinstance(location, basestring):
+        pytest.sel.force_navigate(location)
+    else:
+        dest = 'All'
+        if location is Vm:
+            dest = 'VMsOnly'
+        elif location is Server:
+            dest = 'Dashboard'
+
```
Comment (mshriver) [inline cfme/tests/webui/test_splitter.py]: Is there a fallback if the destination doesn't exist for a location object? Ex if VMsOnly isn't implemented, do we want to fall back to Dashboard, or skip the test?

Comment (izapolsk) [inline cfme/tests/webui/test_splitter.py]: the test fails in such case and that's expected.

**Comment by izapolsk** [comment]
all the issues in tests are fixed and green when I run them locally. But there can be some test failures in PRT due to overall test instability.

**Pull request merged**